source/_integrations/androidtv_remote.markdown

```diff
@@ -44,6 +43,132 @@ Examples of URLs to pass as activity for some popular apps:
 | Prime Video | https://app.primevideo.com
 | Disney+ | https://www.disneyplus.com
 
+Examples:
+
+```yaml
+# Launch the Netflix app
+service: media_player.play_media
+data:
+  media_content_type: url
+  media_content_id: https://www.netflix.com/title
+target:
+  entity_id: media_player.living_room_tv
+```
+
+```yaml
+# Open a specific YouTube video:
+service: media_player.play_media
+data:
+  media_content_type: url
+  media_content_id: https://www.youtube.com/watch?v=dQw4w9WgXcQ
+target:
+  entity_id: media_player.living_room_tv
+```
+
+### Switch channels
+
+You can pass the channel number to switch the channel. The channel number must be an integer.
+
+Example:
+
+```yaml
+# Change channel to number 15:
+service: media_player.play_media
+data:
+  media_content_type: channel
+  media_content_id: 15
+target:
+  entity_id: media_player.living_room_tv
+```
+
+## Remote
+
+The remote allows you to send key commands to your Android TV device with the `remote.send_command` service.
+The entity has the `current_activity` attribute that shows the current foreground app on the Android TV.
+
+{% details "List of the most common commands" %}
+
+Navigation:
+- DPAD_UP
+- DPAD_DOWN
+- DPAD_LEFT
+- DPAD_RIGHT
+- DPAD_CENTER
+- BUTTON_A
+- BUTTON_B
+- BUTTON_X
+- BUTTON_Y
+
+Volume Control:
+- VOLUME_DOWN
+- VOLUME_UP
+- VOLUME_MUTE
+- MUTE
+
+Media Control:
+- MEDIA_PLAY_PAUSE
+- MEDIA_PLAY
+- MEDIA_PAUSE
+- MEDIA_NEXT
+- MEDIA_PREVIOUS
+- MEDIA_STOP
+- MEDIA_RECORD
+- MEDIA_REWIND
+- MEDIA_FAST_FORWARD
+
+TV Control:
+- 0
+- 1
+- 2
+- 3
+- 4
+- 5
+- 6
+- 7
+- 8
+- 9
+- DEL
+- ENTER
+- CHANNEL_UP
+- CHANNEL_DOWN
+- F1
+- F2
+- F3
+- F4
+- F5
+- F6
+- F7
+- F8
+- F9
+- F10
+- F11
+- F12
+- TV
+- PROG_RED
+- PROG_GREEN
+- PROG_YELLOW
+- PROG_BLUE
+
+Other:
+- BUTTON_MODE
+- EXPLORER
+- MENU
+- INFO
+- GUIDE
+- TV_TELETEXT
+- CAPTIONS
+- DVR
+- MEDIA_AUDIO_TRACK
+- SETTINGS
+- SEARCH
+- ASSIST
+
+{% enddetails %}
+
+For a full list see [here](https://github.com/tronikos/androidtvremote2/blob/main/src/androidtvremote2/remotemessage.proto#L90).
```

Comment:
We should not link to source code, please remove this.

Reply:
This was before my changes so I left it. 
In the new PR, I removed it: https://github.com/home-assistant/home-assistant.io/pull/27308